docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
-     cookie=0xc00001100000008, duration=271.027s, table=65, n_packets=0, n_bytes=0, idle_age=271, priority=30,arp,metadata=0x700000000000008/0xff000000ffffffff,arp_op=1 actions=CONTROLLER:65535
-     cookie=0xc00001100000005, duration=272.126s, table=65, n_packets=0, n_bytes=0, idle_age=272, priority=30,arp,metadata=0x700000000000005/0xff000000ffffffff,arp_op=1 actions=CONTROLLER:65535
-
-
-Check Virtual Network Operation.
-------------------------------------
-
-### Start VM(s)
-
-We use kvm to start four virtual machines. Their interfaces were registered in the database earlier. These VMs and their networks are arranged as follows.
-
-+ vm1, vm3 -> vnet1
-+ vm2, vm4 -> vnet2
-
-The command "run-vm-all" starts all four virtual machines at once.
-
-    # cd /root/vnet-test-kvm/vm
-    # ./run-vm-all
-    *** vm1 ***
-    node-id: 1
-    interface name: if-x5vjmksl
-    mac address: 52:54:00:0d:84:8c
-    *** vm2 ***
-    node-id: 2
-    interface name: if-sve1s6xp
-    mac address: 52:54:00:9a:cc:00
-    *** vm3 ***
-    node-id: 3
-    interface name: if-bvixz40n
-    mac address: 52:54:00:0a:fb:78
-    *** vm4 ***
-    node-id: 4
-    interface name: if-7st2alal
-    mac address: 52:54:00:e5:f9:fb
+OpenVNet writes its logs in the /var/log/openvnet directory. If there's a problem starting any of the services, you can find its log files there.
 
-### Check the IP address of virtual machines
+If it launches vna successfully you can see `is_connected: true` by `ovs-vsctl show` such as following.
 
-You can log into the virtual machines using the telnet command.
+```
+fbe23184-7f14-46cb-857b-3abf6153a6d6
+    Bridge "br0"
+        Controller "tcp:127.0.0.1:6633"
+            is_connected: true
+```
 
-    # telnet localhost PORT-NO
+This means the OpenFlow controller which is vna is now connected to the datapath. After the connection between the OpenFlow controller and the
+datapath is established it starts installing the flows on the datapath. You can see the flows by `vnflows-monitor`.
 
-The port numbers are:
+```
+# cd /opt/axsh/openvnet/vnet/bin
+# ./vnflows-monitor
+```
 
-+ vm1 : 5001
-+ vm2 : 5002
-+ vm3 : 5003
-+ vm4 : 5004
+It may require you to setup `PATH` environment variable to find ruby binary. OpenVNet uses its own ruby binary which is in `/opt/axsh/openvnet/ruby/bin` directory.
 
-The ip addresses are assigned as follows. As you can see, vm1 and vm3, share the same ip address, as do vm2 and vm4.
+Verification
+-------
 
-+ vm1 : 10.102.0.10
-+ vm2 : 10.102.0.10
-+ vm3 : 10.102.0.11
-+ vm4 : 10.102.0.11
+By following all the instructions in the previous section, you already have the simple OpenVNet environment. In this section we are going to see network packets going through OpenFlow rules to reach
+from one LXC guest to another.
 
-Log into the virtual machines and make sure their ip addresses are correct.
+Create 2 LXC guests
 
-**ex) vm1**
+```
+# lxc-create -t centos -n inst1
+# lxc-create -t centos -n inst2
+```
```

Comment:
This should be at the very top before we install OpenVNet. Also the reader might not know LXC. We should have a brief explanation of what it is and why we're using it. Then a line that tells them to install LXC through yum.